src/components/ResourcesSection/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Col, Row } from "antd";
+import "./index.css";
+import "antd/dist/antd.css";
+import { Link, BrowserRouter as Router } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import recycleImg from "./../../images/recycle.svg";
+import earthImg from "./../../images/earth.svg";
+import pollutionImg from "./../../images/pollution.svg";
+
+const ResourcesSection = () => {
+  const [t] = useTranslation();
+
+  return (
+    // "site-card-wrapper" is a built in ant Designe className
+    <div className="site-card-wrapper">
+      <Router>
+        <Row justify="center" gutter={[16, 16]}>
+          <Col xxl={6} sm={24}>
+            <Row justify="center">
+              <Col span={22} className="site-card-wrapper__cardName">
```

Comment:
You may rename the className without dashes. 
e.g `siteCard__wrapper__cardName`
For the class names in line 26-35-40-49-54, you can rearrange them as well.